src/pages/sc-cycle-counts/Forms/CycleCountsForm.js

```diff
@@ -212,7 +212,7 @@ export default function CycleCountsForm({ labels, maxAccess: access, setStore, s
     })()
   }, [])
 
-  const disable = formik.values.status === 3
+  const disable = formik.values.status === 3 || isClosed
```

Comment:
why do we need disable?? 
we can only use isposted and isclosed and editmode and set our consitions according to them as a defined states